server.py

```diff
@@ -13,54 +13,106 @@
 apm = ElasticAPM(app, logging=True)
 cors = CORS(app, resources={r"/*": {"origins": "*"}})
 
-"""Request health status of the api"""
+def __is_session_valid(email, session_token):
+    if email and session_token:
+        response = betaAuth.check_session(email, session_token)
+        if response['session'] == True:
+            return True
+        else:
+            return False
+    else:
+        return False
+
 @app.route('/status')
 def health_check():
+    """Request health status of the api"""
     return Response("OK", status=200)
 
-"""Takes a single tweet_id string and returns a list of retweeter id strings"""
-@app.route('/twitter/get_retweeters/<string:tweet_id>')
-def twitter_get_retweeters(tweet_id):
-    return jsonify(twitterApi.get_retweeters(tweet_id))
-
-"""Takes a single tweet_id string and returns a tweet_info json object"""
-@app.route('/twitter/get_tweet_info/<string:tweet_id>')
-def twitter_get_tweet_info(tweet_id):
-    return jsonify(twitterApi.get_tweet_info(tweet_id))
-
-@app.route('/twitter/get_tweet_data/<string:tweet_id>')
-def twitter_get_tweet_data(tweet_id):
-    return jsonify(twitterApi.get_tweet_data(tweet_id))
-
-@app.route('/twitter/get_user_timeline/<string:user_id>')
-def twitter_get_user_timeline(user_id):
-    return jsonify(twitterApi.get_user_timeline(user_id))
-
-"""Takes a comma seperated list of user_ids
+@app.route('/twitter/get_tweet_info', methods = ['POST'])
+def twitter_get_tweet_info():
+    """
+    Returns shortform of twitter_get_tweet_data
+    to get e.g. the number of retweets for a tweet
+    """
+    body = request.get_json()
+    session_token = body['session_token']
+    email = body['email']
+    tweet_id = body['tweet_id']
```

Comment:
Lines 38-40 potentially lead to a lot of `KeyErrors` when any of these fields are not set in json:

![bildschirmfoto vom 2018-11-01 18-36-48](https://user-images.githubusercontent.com/15818147/47868676-c025cd00-ddfc-11e8-8700-c3b7c7920e6e.png)

I suggest using something like 

 ```python
    session_token = body.get('session_token', None)
    email = body.get('email', None)
    tweet_id = body.get('tweet_id', None)
```

Then, if any required parameter is missing (`None`) a `Response("Bad Request", status=400)` should be returned.